src/woo_search/search_index/api/serializers/search.py

```diff
@@ -69,6 +69,14 @@ class SearchSerializer(serializers.Serializer):
         required=False,
         help_text=_("Filter documents that were created before the given value."),
     )
+    informatie_categorieen = serializers.ListField(
+        child=serializers.CharField(),
+        default=[],
+        allow_empty=True,
+        help_text=_(
+            "Filter records based on the given information categories `uuid`."
+        ),
+    )
```

Comment:
Make sure that this filter can't be used if we want to index on Documents.
Also make sure to automatically set the index to "publication" in the es.search func